PurchasesCoreSwiftTests/LocalReceiptParsing/DataConverters/ISO3601DateFormatterTests.swift

```diff
@@ -15,10 +15,26 @@ class ISO3601DateFormatterTests: XCTestCase {
                                             minute: 36,
                                             second: 40)
         let date = Calendar.current.date(from: dateComponents)
-        guard let dateBytes = "2020-07-14T19:36:40+0000".data(using: .ascii) else { fatalError() }
+        guard let dateBytes = "2020-07-14T19:36:40Z".data(using: .ascii) else { fatalError() }
```

Comment:
this isn't really necessary, and the code will work in either format, but it resembles the actual strings in receipts a little more closely